internal/fs/inode/dir_test.go

```diff
@@ -512,7 +513,7 @@ func (t *DirTest) LookUpChild_TypeCaching() {
 
 func (t *DirTest) LookUpChild_NonExistentTypeCache_ImplicitDirsDisabled() {
```

Comment:
Can we also write a reasoning, why we haven't added unit test corresponding to added new flag in the dir_test.go?